example_audit.rules

```diff
@@ -88,20 +88,46 @@ w /etc/fluent/ -p wa -k syslog
 -w /etc/cron.weekly/ -p wa -k cron
 -w /etc/crontab -p wa -k cron
 -w /var/spool/cron/root -p wa -k cron
--w /var/spool/atspool
--w /etc/at.allow
--w /etc/at.deny
+-w /var/spool/cron -p wa -k cron
+-w /var/spool/atspool -p wa -k cron
+-w /etc/at.allow -p wa -k cron
+-w /etc/at.deny -p wa -k cron
+-w /etc/anacrontab -p wa -k cron
  
 # Modify startup
+-w /boot -p wa -k init
 -w /etc/rc.d/init.d/ -p wa -k init
 -w /sbin/init -p wa -k init
 -w /etc/inittab -p wa -k init
 -w /etc/systemd/ -p wa -k init
+-w /etc/systemd/system -p wa -k init
```

Comment:
/etc/systemd should be recursive already

Reply:
Removed